source/frontend/src/components/layout/Healthcheck/HealthcheckContainer.tsx

```diff
@@ -30,39 +30,63 @@ export const HealthcheckContainer: React.FunctionComponent<IHealthcheckContainer
   const keycloak = useKeycloakWrapper();
 
   const checkExternalSystemStatus = useCallback(
-    (response: IHealthcheckResponse, service: HealthcheckMessagesTypesEnum) => {
+    (
+      response: IHealthcheckResponse,
+      service: HealthcheckMessagesTypesEnum,
+      issues: IHealthCheckIssue[],
+    ) => {
```

Comment:
nit: I know this PR is mostly about fixing bugs but I see some inconsistent naming here. We should pick either "IHealthcheck..." or "IHealthCheck..." - I personally prefer the 2nd option. But regardless of which option you pick, we should use a consistent naming